webpack.config.js

```diff
@@ -1,46 +1,59 @@
-const path = require('path');
-const CopyPlugin = require('copy-webpack-plugin');
+const path = require("path");
+const CopyPlugin = require("copy-webpack-plugin");
 
 module.exports = {
-  mode: 'development',
-  entry: './src/index.jsx',
-  devtool: 'eval-source-map',
+  mode: "development",
+  entry: "./src/index.jsx",
+  devtool: "eval-source-map",
   output: {
-    path: path.join(__dirname, 'dist'),
-    filename: 'bundle.js',
+    path: path.join(__dirname, "dist"),
+    filename: "bundle.js",
   },
   devServer: {
     historyApiFallback: true,
-    contentBase: path.join(__dirname, 'dist'),
+    contentBase: path.join(__dirname, "dist"),
     writeToDisk: true,
   },
   resolve: {
-    extensions: ['.js', '.jsx'],
+    extensions: [".js", ".jsx"],
   },
   module: {
     rules: [
       {
         test: /\.jsx?$/,
         exclude: /node_modules/,
         use: {
-          loader: 'babel-loader',
+          loader: "babel-loader",
           options: {
-            presets: ['@babel/preset-env', '@babel/preset-react'],
+            presets: ["@babel/preset-env", "@babel/preset-react"],
           },
         },
       },
       {
         test: /\.css$/,
-        use: ['style-loader', 'css-loader'],
+        use: ["style-loader", "css-loader"],
+      },
+      {
+        test: /\.s[ac]ss$/,
+        use: [
+          "style-loader",
+          "css-loader",
+          {
+            loader: "sass-loader",
+            options: {
+              additionalData: '@import "./src/style.scss";',
```

Comment:
jak jsem psal výše, dal bych si ty variables do separátního souboru, ten importoval tady, a ty "globální styly" normálně importovat jako se už děje.
Když to máš takhle tak se ti ty globální styly zduplikují pro každý scss soubor, a to nechceme :D (myslím že to tak je, ale musel bych si ověřit)

Reply:
To jsem asi moc nepochopila, Tome. Myslíš změnit import ve webpack.config.js a dát tam jen ty variables, které potřebuji globálně a do index.jsx naimportovat style.scss?
![Screenshot 2022-03-30 151436](https://user-images.githubusercontent.com/70709389/160843069-2c277cc6-27ee-4804-9dbd-6542d1447a24.png)

Reply:
jj takhle je to správně 👍 (=tak jsem to myslel)
"additionalData" vloží něco do každého sass souboru. Pokud jsi tam měla @import "./src/style.scss";, tak se jeho obsah přidal do každého sass souboru, což nechceme (duplikovala by se pravidla pro html, ...). 

Takže když do každého souboru automaticky vkládáme jen proměnné, tak to je tip ťop